src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -30,7 +30,8 @@
   "ExternalBlobStore": {
     "BlobContainerUri": null,
     "StorageDirectory": "DICOM/",
-    "HealthCheckFileName": "healthCheck/health.txt"
+    "HealthCheckFilePath": "healthCheck/health",
+    "HealthCheckFileExpiryInMs": 3600000 // 1 hour
```

Comment:
Oh `// 1 hour` was added, not deleted. I think its red because it's not the correct formatting for appsettings.json. I actually will remove it since its showing up red